Tests/MessagingInApp/MessagingInAppImplementationTest.swift

```diff
@@ -188,4 +205,29 @@ class MessagingInAppImplementationTest: UnitTest {
         XCTAssertEqual(eventListenerMock.messageActionTakenReceivedArguments?.actionValue, givenAction)
         XCTAssertEqual(eventListenerMock.messageActionTakenReceivedArguments?.actionName, givenName)
     }
+
+    func test_inAppTracking_givenCustomAction_expectBQTrackInAppClicked() {
```

Comment:
It might be a good idea to add another test function that confirms that when the gist close action occurs, we do not add an item to the BQ.

Reply:
Added that as part of the current test for close, which ensures `BQ` doesn't get updated.